components/Faq/Faq.tsx

```diff
@@ -1,36 +1,72 @@
-import { Container, Stack } from "../bootstrap"
+import { Container, Stack, Col } from "../bootstrap"
 import { FaqQandA } from "./FaqQandA"
 import { FaqCard } from "./FaqCard"
 import styles from "./Faq.module.css"
-import content from "./faqContent.json"
+import { faqContentCards } from "./faqContent"
+import { DropdownButton } from "../shared/DropdownButton/DropdownButton"
+import { useMediaQuery } from "usehooks-ts"
+import { stylingProps } from "../shared/DropdownButton/DropdownButton"
 
 export const Faq = () => {
-  const faqData: FaqData = content
-  const faqKeys: string[] = Object.keys(faqData)
+  const mdMobile = useMediaQuery("(max-width: 768px)")
+  const xsMobile = useMediaQuery("(max-width: 418px)")
 
-  interface FaqData {
-    [key: string]: string | FaqDataCard | any
+  const faqCardHeadings: Array<string> = faqContentCards.map(key => {
+    return key.heading
+  })
+
+  interface Headings {
+    id: number
+    itemName: string
+  }
+
+  interface FaqContentCards {
+    [key: string]: string | FaqContentCard | any
   }
 
-  interface FaqDataCard {
+  interface FaqContentCard {
+    id: number
     heading: string
     qAndA: { question: string; answer: string }[]
   }
 
+  const header1 = "FAQ"
+  const dropdownTitle = "All FAQ Topics"
+
+  const styling: stylingProps = {
+    width: { desktop: "418px", mobile: xsMobile ? "100%" : "418px" }
+  }
+
   return (
-    <>
-      <Container fluid className={styles.faqLayout}>
-        <h1>FAQ</h1>
-        <Stack direction="vertical" gap={4}>
-          {faqKeys.map((key, index) => (
-            <FaqCard
-              key={index}
-              heading={faqData[key].heading}
-              qAndAs={faqData[key].qAndA}
-            ></FaqCard>
-          ))}
-        </Stack>
-      </Container>
-    </>
+    <Container fluid className={styles.faqLayout}>
+      <Stack
+        className={styles.faqHeader}
+        direction={mdMobile ? "vertical" : "horizontal"}
+        gap={mdMobile ? 0 : 4}
+      >
+        <h1>{header1}</h1>
+        <Col xs={xsMobile ? 12 : !12}>
+          <DropdownButton
+            key={dropdownTitle}
+            title={dropdownTitle}
+            styling={styling}
+            variant="secondary"
+          >
+            {faqCardHeadings}
+          </DropdownButton>
+        </Col>
+      </Stack>
+
```

Comment:
```suggestion
import { Row } from "../bootstrap"
import { FaqCard } from "./FaqCard"
import styles from "./Faq.module.css"
import { faqContentCards } from "./faqContent"
import { DropdownButton } from "../shared/DropdownButton/DropdownButton"

export const Faq = () => {
  const faqCardHeadings: Array<string> = faqContentCards.map(key => {
    return key.heading
  })

  const header1 = "FAQ"
  const dropdownTitle = "All FAQ Topics"

  return (
    <Container fluid className={styles.faqLayout}>
      <Row className={styles.faqHeader}>
        <Col xs="auto">
          <h1>{header1}</h1>
        </Col>
        <Col xs={12} sm={6} className="align-self-center">
          <DropdownButton
            key={dropdownTitle}
            title={dropdownTitle}
            variant="secondary"
          >
            {faqCardHeadings}
          </DropdownButton>
        </Col>
      </Row>

```

This uses a Row and Col to set the dropdown size, and avoid needing useMediaQuery

Reply:
Did you pull this pr into your ide and check to see if it worked? When developing I tried this method already but it didn't work. If you try it out you'll see that the dropdown is too big for responsive view. 

The use of useMediaQuery is needed as it prevents the overflow. Those details are super important!

Reply:
when I deployed your code, the size of the dropdown was increasing/decreasing in size as the viewport size changed, also, in the iphone SE view the size of the dropdown does not have 100% width, neither of these should happen as per the figma, the below screen recording is from your deployment   

https://github.com/codeforboston/maple/assets/101157878/6ad2adc1-4703-4353-a567-0a526259d9a5